ui/src/onboarding/containers/LoginPage.tsx

```diff
@@ -9,19 +9,15 @@ import LoginPageContents from 'src/onboarding/containers/LoginPageContents'
 export const LoginPage: FC = () => (
   <ErrorBoundary>
     <AppWrapper className="sign-up--page">
-      <Page titleTag="Sign Up for InfluxDB Cloud">
-        <Page.Contents
-          scrollable={true}
-          fullWidth={true}
-          className="sign-up--page-contents"
-        >
-          <h2 className="cf-funnel-page--title">
-            Create your Free InfluxDB Cloud Account
-          </h2>
-          <h5 className="cf-funnel-page--subtitle">No credit card required</h5>
-          <LoginPageContents />
-        </Page.Contents>
-      </Page>
+      <FunnelPage
+        logo={<InfluxDBCloudLogo cloud={true} className="login-page--logo" />}
+      >
+        <h2 className="cf-funnel-page--title">
```

Comment:
I think as of `2.0.3` these can be a `Heading` component again

Reply:
Good to know! I'll go ahead and make the change